src/components/CallButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+
+const CallButton = () => {
+  return (
+    <StyledCallButton >
```

Comment:
What happens onClick?

Reply:
It's meant to direct calling to a phone number. 
If there is a phone number available, i can add that.